packages/core/src/components/Table/Head/HeadCell/HeadCell.styled.tsx

```diff
@@ -126,7 +126,7 @@ export const HeadCellButton = styled.button<{
     padding: ${({ tableSize }) => resolveValueByTableSize(tableSize, headCellButtonTableSizeMap)};
     outline: unset;
     font-family: inherit;
-    border-radius: 0.8rem;
+    border-radius: ${({ theme }) => theme.table.header.cell.borderRadius ? theme.table.header.cell.borderRadius : '0.8rem'};
```

Comment:
Add the default value in the theme itself so that you don't have to check over here.

Reply:
Haha brain fart